lte/cloud/go/services/subscriberdb/servicers/southbound/subscriberdb_servicer.go

```diff
@@ -110,12 +112,19 @@ func (s *subscriberdbServicer) Sync(
 		return &lte_protos.SyncResponse{Resync: true}, nil
 	}
 
-	resync, renewed, deleted, err := s.getSubscribersChangeset(networkID, req.LeafDigests, digestTree.LeafDigests)
+	resync, renewed, deleted, err := s.getSubscribersChangeset(ctx, networkID, req.LeafDigests, digestTree.LeafDigests)
 	if err != nil {
 		return nil, err
 	}
 	if resync {
-		return &lte_protos.SyncResponse{Resync: true}, nil
+
+		if cloudflag{
+			return nil, nil
+		} else {
+			return &lte_protos.SyncResponse{Resync: true}, nil
+		}
+
+
```

Comment:
I recommend to run `./build.py -c` under orc8r/cloud/docker to run `gofmt` on every Go file being updated here